### ISSUE TITLE
[Accessibility] Narrator cannot announce the check/uncheck state of ListView item when enabling listView checkBox property

### ISSUE THREAD

**Issue by Olina-Zhang** [dotnet/winforms#4029]
* .NET Core Version: 
.Net SDK 5.0.100-rc.2.20477.2
* Have you experienced this same bug with .NET Framework?: 
No
* More info
This is a regression issue, cannot repro in .Net Core 3.1

**Minimal repro:**
1.	Open the attached WinFormsApp9.zip application in VS, then build and run it.
[WinFormsApp9.zip](https://github.com/dotnet/winforms/files/5302826/WinFormsApp9.zip)
2.	Launch Narrator to focus on each ListView items

**Actual behavior:**
**Narrator**: it cannot announce the check/uncheck state of ListView item when enabling listView checkBox property
**Inspect:** Missing Toggle.ToggleState property, and IsTogglePattermAvailable property is false
![image](https://user-images.githubusercontent.com/26474449/94636634-1d3b9d80-0308-11eb-89a6-86b8f2840ff7.png)
![image](https://user-images.githubusercontent.com/26474449/94636645-23317e80-0308-11eb-83a1-3a0281b3817e.png)
**Expected behavior:**
Narrator: it announce the check/uncheck state of ListView item as on/off when enabling listView checkBox property
Inspect: it has Toggle.ToggleState property and supports IsTogglePattermAvailable property
![image](https://user-images.githubusercontent.com/26474449/94636865-c5516680-0308-11eb-83ca-4017cd793594.png)
![image](https://user-images.githubusercontent.com/26474449/94636896-d4381900-0308-11eb-9600-07b4ba3bba1b.png)

**Comment by vladimir-krestov** [comment]
Merged to master branch

**Comment by John-Qiao** [comment]
Verified the issue in Dev16 16.9.0 Preview 1.0 [30613.235.main] + .NET 5.0.100-rtm.20513.7 from 5.0RTM branch , this issue has been fixed, please check the follow screenshots:

1. Narrator: it announce the check/uncheck state of ListView item as on/off when enabling listView checkBox property.
![Narrator](https://user-images.githubusercontent.com/45864985/95952373-8d7b1080-0e2a-11eb-8701-a91967076707.png)

2. Inspect: it has Toggle.ToggleState property and IsTogglePattermAvailable property is true.
![Inspect](https://user-images.githubusercontent.com/45864985/95952390-95d34b80-0e2a-11eb-8a0b-3ea1118784fa.png)
